### DISCUSSION TITLE
Any plans to support Falcon?

### DISCUSSION THREAD

**Discussion by tjedwards** [vllm-project/vllm#197] [Q&A]
While there is news that Meta may be opening models for commercial use soon, currently the Falcon models are some of the best in that space.

**Comment by zhuohan123** [comment]
Supporting Falcon is on our plan! Stay tuned and let us know if you are interested in contributing.

**Reply by rmihaylov** [reply]
Hi, great work! Do you have ETA for Falcon support?
